### ISSUE TITLE
Lakáskód és dallam megváltoztatása

### ISSUE THREAD

**Issue by dobrosi** [dobrosi/jozsefutca#51]
Ebben a dokisban leirtakat majd tesztelni kell:

https://katekaputelefon.hu/wp-content/uploads/2016/09/Kulcsos_kozponti_tajekoztato_lakoknak.pdf

Tudni kell ezeket a dolgokat a mi kaputelefonokkal is vezerelni. (lehet mar mukodik is)

Codefone-nal is ha van ilyen. (gondolom valami van)

**Comment by dobrosi** [comment]
Lakáskód és dallam megváltoztatása
Az átprogramozáshoz két személy kell.
 FELCSENGETÉS (Felcsöngetünk és megvárjuk, hogy fel is vegyék a kaputelefont.)
 CSENGŐ (Lent kétszer megnyomjuk a csengő gombot.)
 CSENGŐ
 KAPUNYITÁS (Felszólunk, hogy fent nyomják meg a kapunyitó gombot.)
 CSENGŐ (Újra megnyomjuk a csengő gombot, sűrű bip jelzi a programozást.)
 KÓD 1. számjegye (Lent beírjuk az új kód 3 számjegyét, de a dallam a hangerő és a)
 KÓD 2. -"- (jelzés számjegyeit is be kell írni az alábbiak szerint. Ha csak a )
 KÓD 3. -"- (dallamot akarjuk változtatni, akkor pedig a régi kódot kell beírni.)
 DALLAM -"- (1-9-ig más-más dallam, 0 véletlenszerűen változó dallam)
 KAPUNYITÁS (Felszólunk, hogy nyomják meg a kapunyitó gombot.)
 CSENGŐ (Megnyomjuk a csengő gombot.)
Ezzel a programozás kész. Célszerű rögtön kipróbálni az új kódot vagy dallamot.
Dallamok: 1. Für Elise 6. A csitári hegyek alatt
2. Kis éji zene 7. Kicsi a hordócska
3. Öröm óda 8. Vége már a szép időknek
4. Ejhaj gyöngyvirág 9. Háromhangú jelzés
5. Hull a pelyhes... 0. A fentiek véletlenszerűen választva.

**Comment by boborjan2** [comment]
codefonnal nincs ilyen, mindent lent kell intezni es a mesterkod kell hozza. gyk csak a szerelo csoinalja, meg aki tudja a kodot. Laskomexnel ugyanez a helyzet.

**Comment by dobrosi** [comment]
Ne mar? Hogy valtozrat halando kapukodot Codefonon?!

**Comment by dobrosi** [comment]
Valszeg tudja csak nem ugy mint a lakotelepin. Nem kell hozza kaputelefonos user.

000
[kulcs]
lakásszám
[kulcs]
régi 4 jegyű kód
új 4 jegyű kód
[kulcs]

**Comment by boborjan2** [comment]
igen, de ehhez programo modban kell lennie a kozpontnak, ahhoz meg kell a szerelokod.

**Comment by dobrosi** [comment]
tudod, vagy gondolod? szerintem nem kell. logikusan ez user use casr.

**Comment by boborjan2** [comment]
ezt tudom. sokat szoptam vele a kazinczy utcaban. Itt is fizetnem kellett a szerelonek, h valtoztassa meg.
Most mar tudom a kodot, szal minden ok.

**Comment by dobrosi** [comment]
Próbáld ki szerintem. Laktam olyan lakotelepi lakasban ahol valtoztattam igy.  Gidofalvi lakas, majd megnezem, de valszeg Codefone.

**Comment by boborjan2** [comment]
de hat pontosan tudom :) itt van egy codefon kozpont a fiokban. Ha eljossz, ki is probalhatod

**Comment by dobrosi** [comment]
Ha tudod a sajat kodod, akkor meg tudod valtoztatni az ismert kodot egy ujra. Te anno nem tudtad, ezert akkor valoban csak mester koddal megy. Azert jo lenne kiprobalni mukodik-e ha a mi belterink log rajta (valoszinu mukodik, mert Codefon kod beallitasnal meg belteri se kell hozza (..elvileg)). A lakotelepin is valszeg fog, mert csak rovidzarakat kell kapcsolgatni. Ha jo lesz a beengeses, teszteljuk! Ezek fontosak, mert alap funkciok nem veszhenek el. Szerencsere tuti menni fog, de csekkoljuk majd....

**Comment by boborjan2** [comment]
A lakotelepiek mashogy mukodnek. Ez a kodprogramozasi lehetoseg is valszeg csak a KATE-val ilyen.
A codefonnal kapcsolatban: minden altalam elerheto info szerint nem lehet megcsinalni, hiaba ismered a sajat kodod. Vagy kuldjel linket vagy doksit, ami alapjan igen.
